doc/source/operating/metrics.rst

```diff
@@ -645,13 +645,16 @@ Reported name format:
 **JMX MBean**
     ``org.apache.cassandra.metrics:type=Client name=<MetricName>``
 
-============================== =============================== ===========
-Name                           Type                            Description
-============================== =============================== ===========
-connectedNativeClients         Gauge<Integer>                  Number of clients connected to this nodes native protocol server
-connections                    Gauge<List<Map<String, String>> List of all connections and their state information
-connectedNativeClientsByUser   Gauge<Map<String, Int>          Number of connnective native clients by username
-============================== =============================== ===========
+============================== ================================ ===========
+Name                           Type                             Description
+============================== ================================ ===========
+connectedNativeClients         Gauge<Integer>                   Number of clients connected to this nodes native protocol server
+connections                    Gauge<List<Map<String, String>>  List of all connections and their state information
+connectedNativeClientsByUser   Gauge<Map<String, Int>           Number of connnective native clients by username
+clientsByProtocolVersion       Gauge<List<Map<String, String>>> List of up to last 100 connections including protocol version. Can be reset with clearConnectionHistory operation in org.apache.cassandra.db:StorageService mbean.
+concurrentRequestsInBytes      Gauge<Long>                      How many concurrent bytes used in current processing requests
+concurrentRequestsInBytesByIp  Histogram                        How many concurrent bytes used in current processing requests by individual ips
```

Comment:
this metric is confusing to me as its not broken down by ip, instead it takes the ip's `RequestsInBytes` and updates a histogram, so more shows the breakdown of bytes concurrently, so better name would be something like `concurrentRequestsInBytesHisto`.

When I read `concurrentRequestsInBytesByIp` I expect `Map<IP, Long>` and showing the amount of bytes broken down by IP